static/beta/prod/navigation/rhel-navigation.json

```diff
@@ -1,318 +1,369 @@
+      "expandable": true,
+      "routes": [
+        {
+          "id": "advisories",
+          "appId": "patch",
+          "title": "Advisories",
+          "href": "/insights/patch/advisories",
+          "product": "Red Hat Insights",
+          "description": "View applicable advisories and updates for your Red Hat Enterprise Linux systems."
+        },
+        {
+          "appId": "patch",
+          "title": "Packages",
+          "href": "/insights/patch/packages",
+          "product": "Red Hat Insights"
+        },
+        {
+          "appId": "contentSources",
+          "title": "Repositories",
+          "href": "/settings/content",
+          "product": "Red Hat Insights"
+        },
+        {
+          "title": "Patch",
+          "expandable": true,
+          "routes": [
+            {
+              "appId": "patch",
+              "title": "Systems",
+              "href": "/insights/patch/systems",
+              "product": "Red Hat Insights"
+            },
+            {
+              "appId": "patch",
+              "title": "Templates",
+              "href": "/insights/patch/templates",
+              "product": "Red Hat Insights"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Operations",
+      "expandable": true,
+      "routes": [
+        {
+          "title": "Advisor",
+          "expandable": true,
+          "routes": [
+            {
+              "id": "recommendations",
+              "appId": "advisor",
+              "title": "Recommendations",
+              "href": "/insights/advisor/recommendations",
+              "product": "Red Hat Insights",
+              "description": "View details about your Red Hat Enterprise Linux systems."
+            },
+            {
+              "id": "advisorSystems",
+              "appId": "advisor",
+              "title": "Systems",
+              "href": "/insights/advisor/systems",
+              "product": "Red Hat Insights",
+              "description": "View details about your Red Hat Enterprise Linux systems."
+            },
+            {
+              "id": "topics",
+              "appId": "advisor",
+              "title": "Topics",
+              "href": "/insights/advisor/topics",
+              "product": "Red Hat Insights"
+            }
+          ]
+        },
+        {
+          "title": "Drift",
+          "expandable": true,
+          "routes": [
+            {
+              "appId": "drift",
+              "title": "Comparison",
+              "href": "/insights/drift/",
+              "product": "Red Hat Insights"
+            },
+            {
+              "appId": "drift",
+              "title": "Baselines",
+              "href": "/insights/drift/baselines",
+              "product": "Red Hat Insights"
+            }
+          ]
+        },
+        {
+          "appId": "policies",
+          "title": "Policies",
+          "href": "/insights/policies"
+        }
+      ]
```

Comment:
mocks show `Resource optimization` here.

Reply:
This latest [mock](https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/mPowdv7) shows it under Business group